web/src/pages/profile/change-password/hooks.js

```diff
@@ -0,0 +1,43 @@
+import { yupResolver } from '@hookform/resolvers/yup'
+import { useRouter } from 'next/router'
+import { useForm } from 'react-hook-form'
+import * as yup from 'yup'
+
+import { useChangePasswordMutation } from '@/hooks/api/authApi'
+import { useHandleError } from '@/hooks/useHandleError'
+
+const schema = yup.object({
+  old_password: yup.string().min(6).required(),
+  password: yup.string().min(6).required(),
+  confirmed_password: yup.string().min(6).required(),
```

Comment:
add below to check if password match

```suggestion
  confirmed_password: yup.string().min(6).required().test('passwords-match', 'Passwords must match', function(value){
      return this.parent.password === value
    }),
```